project/program/views.py

```diff
@@ -1,3 +1,11 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from django.views import View
+from .models import Presenter
 
-# Create your views here.
+class SpeakersView(View):
+
+    template = '../templates/program/speakers.html'
+
+    def get(self, request, *args, **kwargs):
+        PresenterList = Presenter.speakers.all()
+        return render(request, self.template, {'Presenters': PresenterList})
```

Comment:
Same thing about lowercase named variables.
Also, the template context variable better be changed to `speakers`.